ui/app/mainui/SplashScreen.qml

```diff
@@ -25,16 +25,16 @@ Item {
         source: Style.svg("status-logo-circle")
     }
 
-    LottieAnimation {
-        anchors.centerIn: parent
-        autoPlay: false
-        loops: LottieAnimation.Infinite
-        quality: LottieAnimation.MediumQuality
-        source: Style.lottie("status_splash")
-        onStatusChanged: {
-            if (status === LottieAnimation.Ready) {
-                start();
-            }
-        }
-    }
+//    LottieAnimation {
```

Comment:
@alexandraB99 it would be nice to add some comments why the code is commented and if it will be used later or not.